pkg/manilaapi/statefulset.go

```diff
@@ -54,12 +56,47 @@ func StatefulSet(
 	//
 	// https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
 	//
+
 	livenessProbe.HTTPGet = &corev1.HTTPGetAction{
 		Path: "/healthcheck",
 		Port: intstr.IntOrString{Type: intstr.Int, IntVal: int32(manila.ManilaPublicPort)},
 	}
 	readinessProbe.HTTPGet = livenessProbe.HTTPGet
 
+	if instance.Spec.TLS.API.Enabled(service.EndpointPublic) {
+		livenessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+		readinessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+	}
+
+	// create Volume and VolumeMounts
+	volumes := append(GetVolumes(manila.GetOwningManilaName(instance), instance.Name, instance.Spec.ExtraMounts), []corev1.Volume{GetLogVolume()}...)
```

Comment:
could be just `append(GetVolumes(manila.GetOwningManilaName(instance), instance.Name, instance.Spec.ExtraMounts), GetLogVolume())`